Bio/bgzf.py

```diff
@@ -446,7 +445,8 @@ def _load_bgzf_block(handle, text_mode=False):
     if expected_crc != crc:
         raise RuntimeError("CRC is %s, not %s" % (crc, expected_crc))
     if text_mode:
-        return block_size, _as_string(data)
+        import codecs
```

Comment:
Might as well put this import at the top of the file (with the other standard library imports, roughly alphabetical)?